src/App.vue

```diff
@@ -783,7 +784,9 @@ onMounted(() => {
   if ("midiOctaveOffset" in storage) {
     midiOctaveOffset.value = parseInt(storage.getItem("midiOctaveOffset")!);
   }
-
+  if ("intervalMatrixIndexing" in storage) {
+    intervalMatrixIndexing.value = parseInt(storage.getItem("intervalMatrixIndexing")!);
```

Comment:
`parseInt(null)` results in `NaN` so that exclamation mark is definitely in the wrong here. (Remember to test after running `window.localStorage.clear()` in the console.)